kube/services/netpolicy/networkpolicy_pidgin_templ.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: networkpolicy-pidgin
+spec:
+  podSelector:
+    matchLabels:
+      app: pidgin
+  ingress:
+    - from:
+      - podSelector:
+          matchLabels:
+            app: pidgin
+      - podSelector:
+          matchLabels:
+             app: revproxy
+      ports:
+         - port: 80
+         - port: 443
+  egress:
+    - to:
+      - podSelector:
+          matchLabels:
+            app: pidgin
+      - podSelector:
+          matchLabels:
+            app: indexd
```

Comment:
Does 'pidgin' need to talk with indexd?  If so, then we does the settings.py only configure peregrine?